src/AWS/Lambda/RuntimeClient.hs

```diff
@@ -153,7 +154,8 @@ sendEventSuccess rcc@(RuntimeClientConfig baseRuntimeRequest manager _) reqId js
     Right () -> return ()
 
 sendEventError :: RuntimeClientConfig -> BS.ByteString -> String -> IO ()
-sendEventError (RuntimeClientConfig baseRuntimeRequest manager _) reqId e =
+sendEventError (RuntimeClientConfig baseRuntimeRequest manager _) reqId e = do
+  logErrorMsg e
   fmap (const ()) $ runtimeClientRetry $ flip httpNoBody manager $ toEventErrorRequest reqId e baseRuntimeRequest
```

Comment:
Now that this function is no longer a one-liner, something like:

```haskell
let request = httpNoBody (toEventErrorRequest reqId e baseRuntimeRequest) manager
void $ runtimeClientRetry request
```
May be clearer.

Reply:
Good point, agreed.